.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create a release
+
+jobs:
+  build:
+    name: Create Release
```

Comment:
Do we want to add an `if: ${{ github.repository == 'cilium/hubble' }}` here to avoid this job being executed on forks?